src/java.base/share/native/libjli/java.c

```diff
@@ -534,6 +579,316 @@ invokeInstanceMainWithoutArgs(JNIEnv *env, jclass mainClass) {
     return 1;
 }
 
+#if defined(J9VM_OPT_CRAC_SUPPORT)
+/**
+ * Get the value of a command line option in the array of command line arguments.
+ * @param[in] optionName The name of the command line option to search for
+ * @param[in] argc The number of command line arguments
+ * @param[in] argv The array of command line arguments
+ * @param[out] error A pointer to an integer for the error code
+ * @return A pointer to the value of the command line option if found, NULL otherwise
+ */
+static char *getCommandLineOptionValue(const char *optionName, int argc, char **argv, int *error) {
```

Comment:
This should return `const char *` (callers should not be modifying the target of the pointer returned). I see no reason why this needs to allocate space for a copy of the option value; then we don't need the `error` parameter, nor would we need to `free()` those strings later.

The type, name and signature and opening brace should be split across three lines:
```c
static const char *
getCommandLineOptionValue(const char *optionName, int argc, char **argv, int *error)
{
```
The same formatting applies to all new functions added.